lib/install/management/inspect.go

```diff
@@ -147,7 +147,7 @@ func (d *Dispatcher) GetDockerAPICommand(conf *config.VirtualContainerHostConfig
 			}
 
 			dEnv = append(dEnv, "DOCKER_TLS_VERIFY=1")
-			info, err := os.Stat(conf.ExecutorConfig.Name)
+			info, err := os.Stat(certpath)
```

Comment:
Looks like --key and --cert are only for input. --cert-path specifies input (if certs exist) AND output, so this change should work fine with --key and --cert